daras_ai_v2/asr.py

```diff
@@ -373,6 +374,21 @@ def google_translate_language_selector(
     )
 
 
+@redis_cache_decorator(ex=settings.REDIS_MODELS_CACHE_EXPIRY)
+def ghana_nlp_translate_target_languages():
+    """
+    Get list of supported languages for Ghana NLP Translation.
+    :return: Dictionary of language codes and display names.
+    Reference: https://translation.ghananlp.org/api-details#api=ghananlp-translation-webservice-api
+    """
+    r = requests.get(
+        "https://translation-api.ghananlp.org/v1/languages",
+        headers=GHANA_API_AUTH_HEADERS,
+    )
+    raise_for_status(r)
+    return r.json().get("languages", {}) or {}
```

Comment:
just this is enough, the `, {}` part is redundant
`r.json().get("languages") or {}`